spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -408,13 +408,23 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     }
   }
   test("Verify rpad expr support for second arg instead of just literal") {
-    val data = Seq(("IfIWasARoadIWouldBeBent", 10), ("తెలుగు", 2))
+    val data = Seq(("IfIWasARoadIWouldBeBent", 50), ("తెలుగు", 2))
     withParquetTable(data, "t1") {
       val res = sql("select rpad(_1,_2) , rpad(_1,2) from t1 order by _1")
       checkSparkAnswerAndOperator(res)
     }
   }
 
+  test("RPAD with character support other than default space") {
+    val data = Seq(("IfIWasARoadIWouldBeBent", 10), ("hi", 2))
+    withParquetTable(data, "t1") {
+      val res = sql(
+        "select rpad(_1,_2,'?'), rpad(_1,_2,'??') , rpad(_1,2, '??'), hex(rpad(unhex('aabb'), 5)), rpad(_1, 5, '??')  " +
```

Comment:
it would be more readable to keep sql like 

```
sql("""
multiline query
"""")
```

Reply:
Changed the SQL to multiline